source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -544,26 +571,29 @@ static void rialto_mse_audio_sink_set_property(GObject *object, guint propId, co
     case PROP_AUDIO_FADE:
     {
         const gchar *audioFadeStr = g_value_get_string(value);
-        double volume;
-        uint32_t duration;
-        int easeTypeInt;
+        double volume = 0.0;
+        uint32_t duration = 0;
+        int easeTypeInt = convertEaseTypeToInt(firebolt::rialto::EaseType::EASE_LINEAR);
 
         if (sscanf(audioFadeStr, "%lf,%u,%d", &volume, &duration, &easeTypeInt) != 3)
         {
-            GST_ERROR_OBJECT(object, "Failed to parse audio fade string");
-            return;
+            GST_WARNING_OBJECT(object, "Failed to parse audio fade string");
```

Comment:
Print the audioFadeStr in the warning, and print the values that we are going to set.